programs/nodeos/main.cpp

```diff
@@ -63,6 +64,23 @@ void initialize_logging()
    auto config_path = app().get_logging_conf();
    if(fc::exists(config_path))
      fc::configure_logging(config_path); // intentionally allowing exceptions to escape
+   else {
+      auto cfg = fc::logging_config::default_config();
+      cfg.appenders.push_back(
+         fc::appender_config( "deep-mind", "dmlog" )
+      );
+
+      fc::logger_config dmlc;
+      dmlc.name = "deep-mind";
+      dmlc.level = fc::log_level::debug;
+      dmlc.enabled = true;
+      dmlc.appenders.push_back("deep-mind");
+
+      cfg.loggers.push_back( dmlc );
+
+      fc::configure_logging(cfg);
+   }
```

Comment:
Could be a function in `dmlog.hpp` that does this to keep all the `"deep-mind"` config together.

Reply:
Will do

Reply:
Added `add_deep_mind_logger` to `deep_mind.hpp` to take care of adding deep mind logger to default config.